### ISSUE TITLE
Buy order fails after 1 successful trade

### ISSUE THREAD

**Issue by rickdeaconx** [yasinkuyu/binance-trader#93]
Buy executes
Sell executes

This error never stops:
Buy order filled... Try sell...
m: Account has insufficient balance for requested action.

I saw you closed an issue that I thought fixed it but hopefully not repeating. Thanks!

**Comment by iBlindBat** [comment]
Yep, I encountered the exact same. Also took ages to begin the trade even at 0.5% profit.

**Comment by elCapitanoh** [comment]
Yep, I encountered the exact same. Also took ages to begin the trade - even at 0.5% profit.

**Comment by linhndd** [comment]
![screenshot_2](https://user-images.githubusercontent.com/35041874/34930386-c7b9e0aa-f9fb-11e7-963e-3c48cb83d613.png)

I found LOT_SIZE error when running it.

**Comment by SpencerBoyd** [comment]
I am also having this exact issue, everything else is working flawlessly!

**Comment by dikkie-dik** [comment]
same issue:
Buy order filled... Try sell...
Buy order filled... Try sell...
m: Account has insufficient balance for requested action.

Sell order executed. no status change

**Comment by dreKion** [comment]
same issue:

`
1.5% profit scanning for SNMETH

...

price:0.00036405 buyp:0.00036406 sellp:0.00036951-bid:0.00036405 ask:0.00037351
price:0.00036405 buyp:0.00036406 sellp:0.00036951-bid:0.00036405 ask:0.00037351
Buy order created id:xxxx, q:28.00000000, p:0.00036406
Buy order created id:xxxx, q:28.00000000, p:0.00036406
m: Account has insufficient balance for requested action.
m: Account has insufficient balance for requested action.
m: Account has insufficient balance for requested action.
m: Account has insufficient balance for requested action.
m: Account has insufficient balance for requested action.
m: Account has insufficient balance for requested action.
m: Account has insufficient balance for requested action.`

**Comment by jmathon** [comment]
Exactly the same issue as dreKion

**Comment by Jagermeister56** [comment]
check your queue, orderer didn't go through. i have the same problem, but i think it's because the prices moves too fast and bot doesn't get order in on time.
